tests/test_unit_tests.py

```diff
@@ -34,7 +37,7 @@
 
 class TestC2paSdk(unittest.TestCase):
     def test_sdk_version(self):
-        self.assertIn("0.55.0", sdk_version())
+        self.assertIn("0.57.0", sdk_version())
```

Comment:
This needs the new build from https://github.com/contentauth/c2pa-rs/pull/1181 to work